templates/base.html

```diff
@@ -33,54 +34,63 @@
 
 <body>
     <!--Sidebar navigation-->
-    <aside class="aside">
-        <div class="aside-nav">
-            <ul class="aside-nav-box aside-element">
+    <div class="sidebar-container sidebar-responsive sidebar-responsive-on" >
+        <nav class="position-fixed sidebar-responsive sidebar-responsive-on">
+            <ul class="sidebar-navigation sidebar-element">
                 <li id="logo">
                     <a href="#">
                     <span>Orange Pi Admin</span>
+                    <span id="logo-responsive">OPiA</span>
                     </a>
                 </li>
-                <li class="aside-hover">
-                    <a href="#">
-                    <i class="fas fa-chart-area"></i>
-                    <span>System monitor</span>
+                <li class="sidebar-link-hover">
+                    <a href="/index">
+                    <i class="fas fa-home"></i>
+                    <span>Home page</span>
                     </a>
                 </li>
-                <li class="aside-hover">
-                    <a href="#">
+                <li class="sidebar-link-hover">
+                    <a href="/monitor/<string:device_id>"> {#DEV_LITE#}
```

Comment:
1 - <string:device_id> is Flask's route internal type identifier, used to specify location and type of address variable. It's not applicable to plain HTML, and router will not recognize it.

2 - System configuration should lead to configuration page, not device monitor. 

3 - Device monitor button/link should be located underneath respective table on home page.

All things considered - this button should be removed or unbound from it's current href link.